recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script:
+    - echo on
+    - if not exist %SCRIPTS% mkdir %SCRIPTS%
+    - if errorlevel 1 exit 1
+    - xcopy /i /v /f %RECIPE_DIR%\\vs2008_patch\\setup_x64.bat %SCRIPTS%\\
```

Comment:
As you correctly pointed out, this should be `copy`. The `\\` at the end should be dropped.